src/Telemetry/Telemetry.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Azure.WebJobs.Logging;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Sql.Telemetry
+{
+    public sealed class Telemetry
+    {
+        internal static Telemetry Instance = new Telemetry();
+
+        private const string EventsNamespace = "azure-functions-sql-bindings";
+        internal static string CurrentSessionId;
+        private TelemetryClient _client;
+        private Dictionary<string, string> _commonProperties;
+        private Dictionary<string, double> _commonMeasurements;
+        private Task _trackEventTask;
+        private ILogger _logger;
+        private bool _initialized;
+        private const string InstrumentationKey = "98697a1c-1416-486a-99ac-c6c74ebe5ebd";
```

Comment:
Just a sanity check. Is this a new ikey?

Reply:
Yes